pennylane/math/quantum.py

```diff
@@ -19,7 +19,7 @@
 from string import ascii_letters as ABC
 
 from autoray import numpy as np
-from numpy import float64
+from numpy import float64  # pylint:disable=wrong-import-order

```

Comment:
We should have a better fix for this.

Reply:
Please run `isort` on this file and remove disabling pylint. It will likely fix the issue. Here is the imports I locally created with `isort`:

```
import functools
# pylint: disable=import-outside-toplevel
import itertools
from string import ascii_letters as ABC

import pennylane as qml
from autoray import numpy as np

from numpy import float64

from . import single_dispatch  # pylint:disable=unused-import
from .matrix_manipulation import _permute_dense_matrix
from .multi_dispatch import (diag, dot, einsum, get_interface,
                             scatter_element_add)
from .utils import allclose, cast, cast_like, convert_like, is_abstract
```

Reply:
It does not work. It is in conflict with CodeFactor
There is [this](https://xanaduhq.slack.com/archives/CA89H1BAN/p1723647734543739) open discussion